src/rational/q/from.rs

```diff
@@ -492,13 +598,16 @@ mod test_from_int_int {
 
     /// Test with zero denominator (not valid -> should lead to an error)
     #[test]
+    // TODO: Should this test case really panic?
+    // If not, it is necessary to implement try_from with a macro
+    #[should_panic]
```

Comment:
I would say yes and leave it. With more personal consideration, this can remain a panic imo

Reply:
ok, I removed the comment